### ISSUE TITLE
[Discussion] Rethinking card design and colors

### ISSUE THREAD

**Issue by oleast** [dotkom/onlineweb-frontend#232]
## The problem

I've had a thing bugging me about some design choices over the last few months. When making pages with the current design model, I've often hit problems with coloring pages, when to add accent colors, and how to give the page 'life'.

The main problem seems to stem from the background colors of the page.
It is way harder to combine colors with gray (#f2f2f2) than white.

I was not planning on attacking this problem, but a comment for a PR brought me to the point of at least writing this. [Comment by @sardred which sparked this](https://github.com/dotkom/onlineweb-frontend/pull/222#issuecomment-485175622)

### Why is this a large problem

This limits us to using Cards for almost every piece of content on the page, and limits our use of colors. Pages can look very much like they are in grayscale, and therefore more boring and lifeless.

It also makes it harder to do things like full page content. You get into on of two situations:
- You put the image or content directly on the gray (#f2f2f2), which can make it look strange, at least if there are many colors. ([not a good example, but it kinda shows it](https://beta.online.ntnu.no/webshop))
- You add the full-page content into a single Card. This leaves more freedom for the content, but also restricts size to withing the margins of the card, and just looks off. (No example, it has not been done as it looks bad).

Having full-page content can be useful for things like login-, error-, or info-pages. And I would very much like for it to be easier for us to do. 

## How do we fix it

I have two current thoughts about how we can do something about it, but noe are very well formed yet.

### Boring one.

Just lightening the background color could help, I have not really tested it. May not solve all the problems, but may solve some?

### The one about making new design choices

I've had a few thoughts, but again, nothing well formed. [There are some thoughts in the comment that sparked this](https://github.com/dotkom/onlineweb-frontend/pull/222#issuecomment-485175622). Figma would be a good way to start this, and trying to work with with a multitude of the pages we have could prove to be useful.

The first thing that came to my mind was to make our background white (#f2f2f2) again. It takes a tiny bit of inspiration from [Kvitteringsskjema](https://kvittering.online.ntnu.no).

Further it works by using the current Cards more as borders, rather than papers, and also removing the shadow. It would also make it easier to add sections of different colors, like the ones in Kvitteringsskjema.

![Screenshot 2019-04-21 18:09:01](https://user-images.githubusercontent.com/14319313/56472634-2660e380-6461-11e9-8da2-b39d6815604f.png)

![Screenshot 2019-04-21 18:09:16](https://user-images.githubusercontent.com/14319313/56472637-2a8d0100-6461-11e9-9108-105e303a332e.png)

**Comment by sardred** [comment]
A good first step could be to lighten the background a bit (not a lot), maybe shift it more towards blue to make it less dead-looking. It would be cool to look into generating some vector art that we fade out in the background too (cough cough abakus cough cough), triangles are cool for example. 

Going all the way white would require us to shift away from the card design in general, as I don't think just having the borders works well enough then. That would mean having to re-design the event list view for example, and probably a bunch of other pages, all-in-all being a lot of work.

In one way that sucks, but in another it would free us from the layout constraints we've put on ourselves with cards/panes. That being said, cards/panes are _very_ easy to work with and layout comparatively, so sticking with some variation of that could be good for this particular site, given what kind of students are producing contents for it. We're ultimately not designers, and it would be a much harder job to make things work. Then again, cards aren't just a look, they end up shaping the layout, and the longer we stick with it, the more outdated we _might_ become.

Having a look at the [material guidelines](https://material.io/design/components/cards.html) could help. The more we stick to those, the more Google-y we look, but I'm sure there are good design choices we can extract from it without losing our own non-existent look and feel. Looking through how they use color and when in various components could help out. See also the different [examples of implementation](https://material.io/design/material-studies/about-our-material-studies.html) and how you can have sorta your own look without being completely Google Docs'd. Note how the app "Fortnightly" uses cards sparingly, while still having clean visual separation between elements. At the same time, note how the "Reply" example uses cards against a colored background with splashed of complimentaries all around, without having to use too much shadow to add depth.

TL;DR: damn rip lol idk

**Comment by oleast** [comment]
My issues have have largely been addressed by color and card changes from @plusk 

Any further changes should be done with the design system.
